addon/services/service-worker-update-notify.js

```diff
@@ -9,16 +9,12 @@ import serviceWorkerHasUpdate from '../utils/service-worker-has-update'
 const configKey = 'ember-service-worker-update-notify'
 
 async function update() {
-  if (Ember.testing) {
-    return;
-  }
-
   const reg = await navigator.serviceWorker.register(
     '{{ROOT_URL}}{{SERVICE_WORKER_FILENAME}}',
     { scope: '{{ROOT_URL}}' },
   );
 
-  reg.update();
+  return reg.update();
```

Comment:
what does this return?

Reply:
It returns a Promise, so I figured it would make sense to return it, as the function is declared `async` and we have that `yield update();` in the polling task, which wouldn't make too much sense if we didn't return a Promise AFAICT.

Reply:
yup, that's sounds like sound logic to me!